quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMapper.java

```diff
@@ -100,6 +100,29 @@ public class PropertyMapper<T> {
         this.namedProperty = namedProperty;
     }
 
+    /**
+     * This is the heart of the property mapping logic. In the first step, we need to find the value of the property and then transform it into a form of our needs.
+     * <p>
+     *
+     * <b>1. Find value</b>
+     * <p>
+     * In preference order we are looking for:
+     * <pre>
+     *  [ {@link #from} ] ---> [ {@link #mapFrom} ] ---> [ {@link #getDefaultValue()} ] ---> [ directly retrieve `to` value if applicable ]
+     * (explicit)      (derived)        (fallback)
```

Comment:
```suggestion
     *  [ {@link #from} ] ---> [ {@link #mapFrom} ] ---> [ {@link #getDefaultValue()} ] ---> [ {@link #to} ]
     * (explicit)     (derived)           (fallback)         (fallback)
```

Might be better to only use the link to `to` as even default value is not returned if it's not specified. 
That might even look better. @shawkins WDYT?
<img width="871" height="228" alt="image" src="https://github.com/user-attachments/assets/52714561-faed-4836-8be4-03aa9003265f" />